maintenance/build-bundle.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 set -e
 
-cat node_modules/@wikia/ad-engine/dist/ad-engine.global.js <(echo) ./dist/ad-products.global.js > ./dist/global-bundle.js
+cat node_modules/@wikia/ad-engine/dist/ad-engine.global.js <(echo) ./dist/ad-products.global.js <(echo) ./lib/prebid.min.js > ./dist/global-bundle.js
```

Comment:
I decided to add prebid.min.js to global-bundle since it's used in mobile-wiki only and given that we'll avoid additional request for prebid library